### DISCUSSION TITLE
What is the KFP version you are running?

### DISCUSSION THREAD

**Discussion by rimolive** [kubeflow/pipelines#10886] [General]
Based on a discussion in the last Pipelines WG meeting, we'd like to ask  all community users to let us know what is the Kubeflow Pipelines version you are running in your environments, including production. This will help us understand where community is using the most and how can we help fix upgrade issues.

**Comment by milosjava** [comment]
1.8.22

**Comment by thesuperzapper** [comment]
[deployKF](https://www.deploykf.org/) only recently released support for Kubeflow Pipelines 2.1.0 _(a [forked version](https://github.com/deployKF/kubeflow-pipelines) with fixes for v1 pipelines)_ in deployKF [0.1.5](https://www.deploykf.org/releases/changelog-deploykf/#0.1.5), and only about a hundred users have migrated to that version so far.

So you can probably count another thousand-ish user for `2.0.0-alpha.7` which was included with all previous versions of deployKF (and Kubeflow 1.7).

**Comment by rimolive** [comment]
I really appreciate your comments, but the idea of a poll is every community user to pick an option in the form from my initial description. :) Please vote in one of the options.

**Reply by thesuperzapper** [reply]
@rimolive You forgot to include the most popular version `2.0.0-alpha.7` (Kubeflow 1.7)

You also forgot to include "other", as another option because "older versions" would imply older than `1.8`.

**Reply by thesuperzapper** [reply]
I think most people who are using Kubeflow 1.7 will accidentally put `1.8.X` when they are actually using `2.0.0-alpha.7` because we did some crazy stuff with the versioning during the transition to V2.

**Reply by rimolive** [reply]
Let me change the poll options. Thank you for the feedback

**Comment by juliusvonkohout** [comment]
@rimolive is this for the Backend or SDK? For example many use the 2.2.0 and 2.0.5 backend with the KFP SDK 1.8.22. Therefore I voted for 1.8.x

**Reply by rimolive** [reply]
Our focus for now is in the backend. I'd say I'm surprised seeing someone using a newer backend with an old SDK release, but I believe it's because of migration issues. I really appreciate if you could describe the scenario.

**Reply by juliusvonkohout** [reply]
We use Kubeflow 1.9 and 1.8.1 so the 2.0.5 and the 2.2.0 backend with v1 pipelines so the 1.8.22 SDK. I still consider V2 pipelines a beta and have to make sure that they really run rootless and have feature parity. With Kubeflow 1.9 i will evaluate them again.
